web/src/components/Pretreatment.vue

```diff
@@ -90,6 +90,13 @@ v-layout.pretreatment-component(row, fill-height)
                     v-icon.pl-1(small, @click="", v-on="on") {{ $vuetify.icons.info }}
                   span {{ problemData.description }}
 
+            v-list-tile.ml-4.my-1.px-0.small-tile(
+                :class="{ active: $router.currentRoute.name === 'Impute Table' }",
+                @click="$router.push({ path: `/pretreatment/${dataset.id}/cleanup/impute` })")
+              v-list-tile-title.small-tile.pl-2
+                v-icon.pr-1.middle {{ $vuetify.icons.tableEdit }}
```

Comment:
Perhaps make an `impute` icon which is `mdi-file-table-box-outline`?

Reply:
we don't have this specific icon in our iconfont

Reply:
Hm, ok @jtomeck may need to rescue us here. I'm ok with keeping this icon for now.

Reply:
@sgratzl Are you saying it isn't available in version 3.5.95, or that it needs to be added in `utils/vuetifyConfig.js`?  I see that icon in @mdi/font so we should be able to use it.

Reply:
there is no CSS class defined in `@mdi/font/materialdesignicons.css` with the name `mdi-file-table-box-outline` in my version 3.5.95

Reply:
Looks like it's in master for mdi, maybe we just `yarn update @mdi/font` and it will be there?

https://github.com/Templarian/MaterialDesign-Webfont/blob/master/css/materialdesignicons.css